### ISSUE TITLE
SSH banner can break scp transfer back to local nodes

### ISSUE THREAD

**Issue by markcurtis1970** [dremio/dremio-diagnostic-collector#102]
If a user is running a remote collect using a SSH connection, the final scp step to pull back the archive to the local machine will fail. This is because we run a `hostname` command to get the hostname to derive the ddc filename under `/tmp/ddc` typically:

```
ip-10-10-10-63.eu-west-1.compute.internal.tar.gz
```

If we have a banner we see this kind of thing in the `ddc.log`

```
ERROR: 2023/08/10 14:45:03 sudo.go:111: host 10.1.2.3 failed to run command with error unable to start command 'scp -i /Users/mc/Support/mc-ssh-ireland.cer -o UserKnownHostsFile=/dev/null -o StrictHostKeyChecking=no ec2-user@10.1.2.3:/tmp/ddc/*******************
Welcome to AWSE!
*******************
ip-10-10-10-63.eu-west-1.compute.internal.tar.gz'
```

**Comment by markcurtis1970** [comment]
Fix is coded waiting on confirmation / testing

**Comment by rsvihladremio** [comment]
fixed in #109
